backend/package.json

```diff
@@ -9,6 +9,9 @@
     "nodemon": "^2.0.22"
   },
   "dependencies": {
+    "cors": "^2.8.5",
+    "crypt": "^0.0.2",
```

Comment:
mistake?

Reply:
No, actually I installed these packages because I was prompted with a CORS error. I used Crypt to generate the hash of the password.